Documentation/ceph-dashboard.md

```diff
@@ -111,3 +111,62 @@ rook-ceph-mgr-dashboard-external-https  NodePort    10.101.209.6     <none>
 In this example, port `31176` will be opened to expose port `8443` from the ceph-mgr pod. Find the ip address
 of the VM. If using minikube, you can run `minikube ip` to find the ip address.
 Now you can enter the URL in your browser such as `https://192.168.99.110:31176` and the dashboard will appear.
+
+If you have a cluster with an [nginx Ingress Controller](https://kubernetes.github.io/ingress-nginx/)
+and a Certificate Manager (e.g. [cert-manager](https://cert-manager.readthedocs.io/)) then you can create an
+Ingress like the one below. This example achieves four things:
+
+1. Exposes the dashboard on the Internet (using an reverse proxy)
+2. Issues an valid TLS Certificate for the specified domain name (using [ACME](https://en.wikipedia.org/wiki/Automated_Certificate_Management_Environment))
+3. Tells the reverse proxy that the dashboard itself uses HTTPS
+4. Tells the reverse proxy that the dashboard itself does not have a valid certificate (it is self-signed)
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: rook-ceph-mgr-dashboard
+  namespace: rook-ceph
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    kubernetes.io/tls-acme: "true"
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+    nginx.ingress.kubernetes.io/server-snippet: |
```

Comment:
Is this really necessary? Although this makes sense, for some reason, for me, the dashboard still works fine without this.

Reply:
@lbogdan Due to the MGR cert being self signed, this is needed as there is currently no annotation (besides this "hack" to do that) to skip cert verify of backend servers.

Reply:
Yes, I totally get the reason why this should be done, I was just saying that with a "default" `nginx-ingress` deployment, the ceph dashboard works fine for me, without explicitly configuring this - which kind of came as a surprise for me.

Reply:
@lbogdan this is required for `nginx-ingress` just because it is default `nginx` behavior and there is not currently an `nginx-ingress` option to disable/enable it. Maybe you weren't using (verified) HTTPS downstream? Had you use the new option to disable HTTPS on the dashboard?